examples/medplum-provider/src/pages/encounter/EncounterComplete.tsx

```diff
@@ -39,28 +61,34 @@ export const EncounterComplete = (): JSX.Element => {
         message: normalizeErrorString(err),
       });
     });
-  }, [medplum, encounterId, fetchTasks, location.pathname]);
+  }, [medplum, encounter, fetchTasks]);
+
+  if (!patient || !encounter) {
+    return <Loading />;
+  }
 
   return (
     <>
-      <Box p="md">
-        <Text size="lg" color="dimmed" mb="lg">
-          Encounter {encounter?.period?.start ?? ''}
-        </Text>
+      <Stack justify="space-between" gap={0}>
+        <EncounterHeader patient={patient} encounter={encounter} practitioner={practitioner} />
+
+        <Box p="md">
+          <Text size="lg" color="dimmed" mb="lg">
+            Encounter {encounter?.period?.start ?? ''}
+          </Text>
 
-        <div style={{ display: 'grid', gridTemplateColumns: '1fr 300px', gap: '24px' }}>
-          <Stack gap="md">
+          <div style={{ display: 'grid', gridTemplateColumns: '1fr 300px', gap: '24px' }}>
             <Stack gap="md">
-              {tasks?.map((task: Task) => <TaskQuestionnaireResponseSummaryPanel task={task} key={task.id} />)}
+              <Stack gap="md">{task && <TaskQuestionnaireResponseSummaryPanel task={task} key={task.id} />}</Stack>
             </Stack>
-          </Stack>
 
-          <Stack gap="lg">
-            <Button variant="outline">View Claim</Button>
-          </Stack>
-        </div>
-        <Outlet />
-      </Box>
+            <Stack gap="lg">
```

Comment:
If you only need vertical padding, you should be able to add `py` attribute to the `<Button>` element.

See here: https://mantine.dev/styles/style-props/